src/workloads/execution/BackgroundIndexConstruction.yml

```diff
@@ -0,0 +1,156 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: |
+  This workload tests the write performance impact of creating indexes on a very large collection.
+  It synthetically lowers the number of concurrent operations allowed to represent a saturated
+  server.
```

Comment:
Could you add something saying we measure and care about the index creation time and the throughput and latency of the inserts and removes of the background latency? Possibly also mention that we may see trade-offs between index creation time and background workload performance.

Reply:
Could you also describe up here that you are both

1. Building an index with a workload against a different collection
2. Building an index with a workload against the same collection

At different points in the workload?

Reply:
Done, I'll proceed to also revert the addition of point 2 as it seems it times out with our depriorisation changes and it would make the test fail with the feature flag enabled.